admin_guide/scheduler.adoc

```diff
@@ -668,3 +668,163 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master
 ----
 endif::[]
+
+[[controlling-pod-placement-projects]]
+=== Control Pod Placement to Projects
+
+The _Pod Node Selector_ admission controller allows you to force pods onto nodes associated with a specific project and prevent pods from being scheduled in those nodes.
```

Comment:
@mburke5678 You use underscores above, but then asterisks below. I'd say neither are needed.